client/src/contracts/deployments.ts

```diff
@@ -1,10 +1,14 @@
 export const chains: { [key: number]: string } = {
+  31337: "localhost",
   5: "goerli",
   10: "optimism",
   69: "optimisticKovan",
 };
 
 export const addresses: { [key: string]: any } = {
+  localhost: {
+    projectRegistry: "0x832c5391dc7931312CbdBc1046669c9c3A4A28d5",
```

Comment:
@ghostffcode is this actually something we want to hardcode?
cause in theory it would be different for everyone right?

Reply:
It's just working off a fork of goerli instead of running a fresh local node and deploying contracts to it.